### PR TITLE
zip file processing modifications

### PR DESCRIPTION
**Pull request by rkazak**

### CODE REVIEW

File: logmerge.py [hunk 1]
```diff
@@ -72,6 +71,7 @@ def main_with_args(args, visitor=None, path_prefix=None, bar=None):
             visitor=visitor,
             wrap=args.wrap,
             wrap_indent=args.wrap_indent,
+            verbosity=args.verbose,
```
Comment (steveyen) [inline logmerge.py]: better, imho, to keep the same name throughout the codebase, if possible... in this case, "verbose" and not rename it to verbosity?

File: logmerge.py [hunk 2]
```diff
@@ -215,20 +213,21 @@ def process(paths,
...
             w=None,                   # Optional output stream.
+            verbosity=0,              # verbosity.
             bar=None):                # Optional progress bar.
     # Find log files.
     paths, total_size, path_sizes = expand_paths(paths, suffix)
 
     if not path_prefix:
         if len(paths) > 1:
-            path_prefix = os.path.commonprefix(paths)
+            path_prefix = paths[0].split(':')
```
Comment (steveyen) [inline logmerge.py]: little worried about this, as i think commonprefix did provide some value here, which we might lose with a switch to ':' splitting

Comment (steveyen) [reply logmerge.py]: for example, I'd worry that non-zip file input might be affected here?

File: logmerge.py (hunk 1 above)
Comment (rkazak) [reply logmerge.py]: Agreed

File: logmerge.py (hunk 2 above)
Comment (rkazak) [reply logmerge.py]: OK, will recheck how better to use that. And other containers could be a feature request!

Comment (rkazak) [reply logmerge.py]: os.path.commonprefix is good as long as the directory structure within the zip file is flat. 
If there are subdirectories within the zip file then os.path.commonprefix returns the string which is the lowest common denominator for the collection of available paths.

**Review by steveyen — APPROVED** [review]

**Pull request merged**